OpenSim/Common/Property.h

```diff
@@ -693,17 +693,16 @@ Property<T>::getTypeName() const {
     return TypeHelper::getTypeName();
 }
 
-// Hide SimpleProperty and ObjectProperty from Doxygen; users don't need
-// to know about these.
-/** @cond **/
-
```

Comment:
Oh oops; I didn't realize this was all hidden. In that case, you could leave these classes as hidden and add the comment about the precision back in the base class.

Sorry about that!